R/rank.R

```diff
@@ -1,8 +1,3 @@
-# TODO: Use this NEWS bullet when we export `vec_rank()`
-#
-# * New `vec_rank()` to compute various types of sample ranks.
-
-
 #' Compute ranks
 #'
 #' `vec_rank()` computes the sample ranks of a vector. For data frames, ranks
```

Comment:
Note that these docs reference `Like [vec_order_radix()]`, i.e. the unexported `vec_order_radix()` function that we do have public documentation for, it is just hidden away with `@keyword internal`.

I think this is fine, when we eventually make `vec_order()` use `vec_order_radix()`, this will just get updated to point to `vec_order()` instead.